### ISSUE TITLE
Effects - update the combat bar info after monster attacks

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#203]
```
The effects symbol likes to stay even after the effect is over or vise versa at 
times also. With fitgue it will show same cost for ap but it will cost an extra 
2 for example my ap attack is 4 but it will use up 6 ap. With the healing for 
the drake trainers does not appear or it is not working.
```

Original issue reported on code.google.com by `dope...@gmail.com` on 2 Jul 2011 at 3:50

**Comment by GoogleCodeExporter** [comment]
```
the healing ability problem is also appearing with all aulaeth.
```

Original comment by `dope...@gmail.com` on 2 Jul 2011 at 4:03

**Comment by GoogleCodeExporter** [comment]
```
Thank you for the report. Yes, this is a bug. The combat bar does not update 
the text on the attack button (containing the number of APs an attack takes) 
after a monster has attacked.

This is now fixed in r121. Marking this as ready for the next release.
```

Original comment by `oskar.wi...@gmail.com` on 8 Jul 2011 at 3:50
- Changed state: **Ready**
- Added labels: **Milestone-0.6.10**

**Comment by GoogleCodeExporter** [comment]
```
Also, thanks for reporting the issue with monsters that heal when attacking. 
This is also a bug (actually, it's the same bug). This is now also fixed by 
r121.

Thank you for reporting it!
```

Original comment by `oskar.wi...@gmail.com` on 8 Jul 2011 at 3:55

**Comment by GoogleCodeExporter** [comment]
Original comment by `oskar.wi...@gmail.com` on 16 Jul 2011 at 10:12
- Changed title: **Effects - update the combat bar info after monster attacks**

**Comment by GoogleCodeExporter** [comment]
```
The problem in the last version where your AP would not show the right amount 
is reappearing again, but this time without fatigue. Another wise the rest of 
old problems are going away. Another problem is when you kill a creature with 
Halberd from Guther you gain one life but when you walk away from combat it 
will show many times that I’m gaining life. When in fact I am not.
```

Original comment by `dope...@gmail.com` on 20 Oct 2011 at 4:58

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
